### PR TITLE
attempt to update to newer version

### PR DESCRIPTION
**Pull request by TnTo**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by TnTo** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,20 +8,17 @@ package:
...
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - pip
-    - python
     - poetry
   run:
-    - flake8 >=3.7,<3.9
+    - flake8 >=3.7
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:22]: According to the metadata of this package, this is necessary
```suggestion
    - flake8 >=3.7,<4.0
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,20 +8,17 @@ package:
...
 build:
   number: 0
-  skip: true   # [py>=40 or py2k]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - pip
-    - python
     - poetry
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:19]: Please, put ``pip`` and ``python back here

**Comment by TnTo** [comment]
@muyajil
It should be possible to merge this PR to get python 3.9 compatibility

**Pull request merged**